src/yang/ietf-babel.yang

```diff
@@ -85,31 +85,49 @@ module ietf-babel {
   feature mac-supported {
     description
       "This implementation supports MAC based security.";
+    reference
+      "draft-ietf-babel-hmac: MAC authentication for Babel Routing
+       Protocol.";
   }
 
   feature dtls-supported {
     description
       "This implementation supports DTLS based security.";
+    reference
+      "draft-ietf-babel-dtls: Babel Routing Protocol over Datagram
+       Transport Layer Security.";
   }
 
   feature hmac-sha256-supported {
     description
       "This implementation supports hmac-sha256 MAC algorithm.";
+    reference
+      "draft-ietf-babel-hmac: MAC authentication for Babel Routing
+       Protocol.";
   }
 
   feature blake2s-supported {
```

Comment:
I think we should keep blake2s to encompass future definitions of blake2-256, blake2-224 and other variations of blake2s that can then be added without having to add a feature statement for each of them.

I will change the description to say:  "This implementation supports blake2s MAC algorithms."

Finally, as far as lowercase/uppercase is concerned, YANG requires all names to be lowercase, but the English language does not, and since it is described with all uppercase letters, it is ok to reference them as such in the description.

Reply:
A comment resolution for the info model IETF LC was to specifically distinguish what size BLAKE2s MAC was being used. So the Babel MAC draft will be updated to specifically call out 128 bit. And it was agreed that the info model would do the same. I'm not sure it's a good idea to make the YANG model different.
Here was Valery's reply to my agreeing to make this change: https://mailarchive.ietf.org/arch/msg/babel/QztS9JBz5gjH_beex1x1_P9Zxqw/
And this was the discussion on the babel email list: https://mailarchive.ietf.org/arch/msg/babel/wax09oIrJTadGfOWAYgCHoTqhkc/

Reply:
Agreed to add "Specifically, BLAKE2-128 is supported."

Reply:
Added.